static/app/utils/theme.tsx

```diff
@@ -692,39 +944,62 @@ const iconSizes = {
   xxl: `${iconNumberSizes.xxl}px`,
 } as const;
 
+// @TODO(jonasbadalic): This was missing profiles, profileChunks, profileDuration, spans, spansIndexed, uptime, what do we do with them?
```

Comment:
Having such a tight coupling to `DataCategory` indicates these aren't general purpose theme utils and should belong in another file imo

Reply:
I want to just completely remove this tbh. I've started removing some other values so I hopefully I get to it soon